modules/rte_mis_lottery/src/Form/LotteryConfigForm.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\rte_mis_lottery\Form;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * RTE MIS Lottery config form.
+ */
+class LotteryConfigForm extends ConfigFormBase {
+
+  /**
+   * Config settings.
+   *
+   * @var string
+   */
+  const SETTINGS = 'rte_mis_lottery.settings';
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The current user.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $currentUser;
+
+  /**
+   * Constructs the service objects.
+   *
+   * Class constructor.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager, AccountInterface $current_user) {
+    $this->entityTypeManager = $entityTypeManager;
+    $this->currentUser = $current_user;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('entity_type.manager'),
+      $container->get('current_user')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'rte_mis_lottery_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      static::SETTINGS,
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config(static::SETTINGS);
+
+    $form['time_interval'] = [
+      '#type' => 'number',
+      '#title' => $this->t('Time Interval'),
+      '#description' => $this->t('Enter the time interval in hours after which the lottery status table will be cleared.'),
+      '#default_value' => $config->get('time_interval'),
+      '#min' => 1,
+      '#required' => TRUE,
+    ];
+
```

Comment:
Is this accepting float value? Can we add validation of minimum value of 24 hours?